views/leaderboard-table.pug

```diff
@@ -1,44 +1,41 @@
-table.bordered
-  thead
-    tr
-      th Rank
-      th User ID
-      th Points
-      if fullTable
-        th Attempts
-        th Correct Answers
-        th Success Rate
-  tbody
-    each s in leaderboard
-      if s.rank == 0
-        tr.col-default
-          td ...
-          td ...
-          td ...
-          if fullTable
-            td ...
-            td ...
-            td ...
-      else
-        if s.points != 0 && s.rank <= 3
-          if s.rank == 1
-            - rowclass = 'col-gold';
-          if s.rank == 2
-            - rowclass = 'col-silver';
-          if s.rank == 3
-            - rowclass = 'col-bronze';
-        else
-          - rowclass = 'col-default';
-        if s.id == userid
-          - rowclass += ' col-isuser';
-        tr(class=rowclass)
-          td #{s.rank}
-          td #{s.id}
-          td #{s.points}
-          if fullTable
-            td #{s.totalAttemptsCount}
-            td #{s.correctAttemptsCount}
-            if s.totalAttemptsCount === 0
-              td 0%
+.row#overall-board-card
```

Comment:
whats the sorting type of this leaderboard?

Reply:
It's sorted by the oveall score, overall score = total points * (accuracy + points per attempt / 100)